generator/src/test/java/com/scottlogic/deg/generator/cucumber/LessThan.feature

```diff
@@ -2,12 +2,332 @@ Feature: User can specify that a numeric value is lower than, but not equal to,
 
 Background:
      Given the generation strategy is full
-      And there is a field foo
-      And foo is of type "numeric"
-      And foo is anything but null
+       And there is a field foo
+       And foo is of type "numeric"
+       And foo is anything but null
```

Comment:
remove anything but null in the background so that soft contradiction logic can be validated with the generation of null data.

Reply:
See later comments but how would this add value to tests around lessThan?

Reply:
An error or no data generation can be caused by the app failing quietly, whereas to produce null data shows that the app has been engaged and the soft contradiction logic is working as expected.